src/applications/vaos/components/FormButtons.jsx

```diff
@@ -15,15 +15,15 @@ export default function FormButtons({
 }) {
   return (
     <div className="vads-l-row form-progress-buttons schemaform-buttons">
-      <div className="xsmall-screen:vads-u-padding-right--1p5 medium-screen:vads-u-padding-right--0p5">
+      <div className="vaos__form-button xsmall-screen:vads-u-padding-right--1p5 medium-screen:vads-u-padding-right--0p5">
```

Comment:
The ticket has these buttons with the Continue on top, but the screenshot in the PR here has the Back button on top.